rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -195,7 +204,7 @@ void getActiveFederatorPublicKeyOfType() {
         assertThrows(IndexOutOfBoundsException.class, () -> federationSupport.getActiveFederatorPublicKeyOfType(-1, KeyType.BTC));
     }
 
-    private Federation getNewFakeFederation(long creationBlockNumber) {
+    private Federation createFederation(long creationBlockNumber) {
```

Comment:
Can we remove this and make use of FederationTestUtils in case? I think it's better we have utility methods centralized and we avoid re-creating them many times.

Also, as much as possible use an ERP Federation for testing. It's more realistic since it's what we have now

Reply:
done